BuyPolish/Pola/SupportingFiles/pl.lproj/Localizable.strings

```diff
@@ -43,6 +43,7 @@
 "mail_title" = "Wiadomość wysłana z aplikacji Pola";
 "Scan barcode" = "Umieść kod kreskowy produktu w prostokącie powyżej aby dowiedzieć się więcej o firmie, która go wyprodukowała.";
 "Type 13 digits" = "Wpisz 8 lub 13 cyfrowy kod produktu";
```

Comment:
Stare tłumaczenie jest nieużywane nigdzie, więc dobrze byłoby je usunąć :)

Reply:
Jasne, dzięki! 

plan snapshots uruchamiam na swoim branchu?

Reply:
Tak na swoim, wtedy do Twojego brancha github zacommituje zmiany.

Reply:
Dzięki za pomoc, dopiero uczę się githuba.

Reply:
To dodatkowy tip: jak w tytule lub w opisie commita dasz Fix/fixes/close/closes i po tym #161 to github powiąże PRa z ticketem automatycznie :)